CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ desktop: fix gas switches in UDDF exports
 core: allow of up to 6 O2 sensors (and corresponding voting logic)
 desktop: add divemode as a possible dive list column
 profile-widget: Now zomed in profiles can be panned with horizontal scroll.
+media: support raw files if libraw is installed
```

Comment:
I think with the move to publishing CICD releases this file is getting to the end of its usefulness...